playbooks/eucalyptus_basic_server.yml

```diff
@@ -25,8 +25,8 @@
       - edx_notes_api
       nginx_default_sites:
       - lms
-    - role: edxlocal
-      when: EDXAPP_MYSQL_HOST == 'localhost'
+    - mysql
+    - mysql_init
```

Comment:
`mysql_init` allows us to set the host for users. `edxlocal` doesn't allow this.